### PR TITLE
Adding support for Cisco ASA banner when nested underneath a parent config

### PR DESCRIPTION
**Pull request by huacchob**
Banner for Cisco ASA could be a child config, and would start with a white space. I added that support to both the BaseSpaceConfigParser.banner_start list and to CiscoConfigParser.regex_banner attribute

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
hey, can you provide test cases for this?

**Comment by huacchob** [comment]
Would this test be enough?

File: netutils/config/parser.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ class BaseSpaceConfigParser(BaseConfigParser):
     # pylint: disable=abstract-method
 
     comment_chars = ["!"]
-    banner_start = ["banner", "vacant-message"]
+    banner_start = ["banner", "vacant-message", " banner", "  banner"]
```
Comment (itdependsnetworks) [inline netutils/config/parser.py:56]: I don't understand this, why are we adding banner_start multiple times?

Comment (huacchob) [reply netutils/config/parser.py:56]: The first one has has 1 white space and the second one has two.

Comment (itdependsnetworks) [reply netutils/config/parser.py:56]: I would not think this is how we want to do this then. I would think we would understand the parent / child relationship would be how we determine it.

Comment (itdependsnetworks) [reply netutils/config/parser.py:56]: Check in with @jmcgill298

Comment (jmcgill298) [reply netutils/config/parser.py:56]: ya, that banner is a child of the group-policy, so it should be captured there. I guess for this case the banner is not multiline?

**Pull request closed without merging**